src/nusoap.php

```diff
@@ -908,11 +908,15 @@ function getmicrotime()
     function varDump($data)
     {
         ob_start();
-        var_dump($data);
+        var_dump($this->sanitize($data));
```

Comment:
`$data` here is not guaranteed to be a string. We also don't have an output context here, so escaping is inappropriate - it might never be displayed in an XML or HTML context.